react/react1/week2/vite-project/src/useFetch.jsx

```diff
@@ -0,0 +1,36 @@
+import { useState, useEffect } from "react";
+
+const useFetch = (url) => {
```

Comment:
Really creative! I'd advice you to use something like SWR (https://swr.vercel.app/) going forward, which is a framework for fetching data 👍

Reply:
thanks for reviewing my homework and I really appreciate that.
and I'm gonna fix the code, now I'm working on adding edit functionality to my todo, using the put method..